site/src/components/WorkspaceActionButton/WorkspaceActionButton.tsx

```diff
@@ -2,20 +2,22 @@ import Button from "@material-ui/core/Button"
 import { FC } from "react"
 
 export interface WorkspaceActionButtonProps {
-  label: string
+  label?: string
   icon: JSX.Element
   onClick: () => void
   className?: string
+  ariaLabel?: string
 }
 
 export const WorkspaceActionButton: FC<WorkspaceActionButtonProps> = ({
   label,
   icon,
   onClick,
   className,
+  ariaLabel,
 }) => {
   return (
-    <Button className={className} startIcon={icon} onClick={onClick}>
+    <Button className={className} startIcon={icon} onClick={onClick} aria-label={ariaLabel}>
       {label}
```

Comment:
Since this is an optional prop now, maybe we should update this to `!!label && label`? Then we can probably not pass the empty string for the Cancel button.

Reply:
what a good catch!